alert/alert.css

```diff
@@ -0,0 +1,145 @@
+@import url(/CSS/nav.css);
+
+:root {
+
```

Comment:
declare root variables in one file and then use that no need to declare it in every CSS file